backend/db/migrate/20220303132723_create_open_calls.rb

```diff
@@ -0,0 +1,38 @@
+class CreateOpenCalls < ActiveRecord::Migration[7.0]
+  def change
+    create_table :open_calls, id: :uuid do |t|
+      t.belongs_to :investor, foreign_key: {on_delete: :cascade}, type: :uuid, null: false, index: true
```

Comment:
Hmm, used to slightly different approach:
 - at db, you just define `foreign_key: true`
 - what should happens with related record is defined on Rails level `has_many :open_calls, dependent: :destroy`
 
Same applies for `null: false` <-- this is by default checked by Rails via `belongs_to :investor, optional: ?`
And I believe that `index: true` is default, no need to write it

Reply:
Never depend on application level validations only. The database has performant and standardised mechanisms to protect integrity of your data. Databases tend to live longer than applications.

Reply:
Valid point @agnessa , thanks ;). I would mention that Rails defines often a lot of hooks on app level, therefore if you delete cascade relations on db level, you avoid these hooks completely. But you are right with your points